src/shims/tls.rs

```diff
@@ -235,7 +235,7 @@ trait EvalContextPrivExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
     fn schedule_windows_tls_dtors(&mut self) -> InterpResult<'tcx> {
         let this = self.eval_context_mut();
         let active_thread = this.get_active_thread();
-        assert_eq!(this.get_total_thread_count(), 1, "concurrency on Windows is not supported");
```

Comment:
So, is this code correct for windows? We have to do the exact same thing for all threads?

Reply:
Yes, this function is called for each thread when it exits. Technically, the reason should be different for the first thread and I don't know if the first and last args need to be different either, but std ignores all of this.

Reply:
Sounds fair, but please add a comment explaining that point about the reason and the other arguments.

Reply:
This is still open, please add such a comment.